### PR TITLE
Reply Adaptive codelab end package

### PR DESCRIPTION
**Pull request by Gurupreet**
END branch for Adaptive codelab related to main PR 

[Start PR](https://github.com/googlecodelabs/android-compose-codelabs/pull/249)

 Based out of sample
 https://github.com/android/compose-samples/tree/Gurupreet/reply-m3 which is full sample app in progress.
 
 This codelab shows the emphasis on adaptability and accessibility part of the sample.
 
 [Codelab doc](https://docs.google.com/document/d/1WNI_T_8RODkIs_0b-mIWSkGixpoWfXAluJN7pXRE8Pg/edit#)

### CODE REVIEW

File: AdaptiveUiCodelab/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Adaptive and accessible apps in Jetpack Compose Codelab
+
+This folder contains the source code for
```
Comment (florina-muntenescu) [inline AdaptiveUiCodelab/README.md:3]: can we mention that this is the solution?

File: AdaptiveUiCodelab/app/src/main/res/drawable-v24/ic_launcher_foreground.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (florina-muntenescu) [inline AdaptiveUiCodelab/app/src/main/res/drawable-v24/ic_launcher_foreground.xml:1]: copyright

File: AdaptiveUiCodelab/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#Fri Mar 04 12:02:05 CET 2022
```
Comment (florina-muntenescu) [inline AdaptiveUiCodelab/gradle/wrapper/gradle-wrapper.properties:1]: copyright

File: AdaptiveUiCodelab/gradlew [hunk 4]
```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
```
Comment (florina-muntenescu) [inline AdaptiveUiCodelab/gradlew:4]: 2022

File: AdaptiveUiCodelab/gradlew.bat [hunk 5]
```diff
@@ -0,0 +1,89 @@
+@rem
+@rem Copyright 2015 the original author or authors.
```
Comment (florina-muntenescu) [inline AdaptiveUiCodelab/gradlew.bat:2]: 2022

**Review by florina-muntenescu — APPROVED** [review]
Copyright missing in xmls and smaller comments, otherwise lgtm

File: AdaptiveUiCodelab/gradlew (hunk 4 above)
Comment (Gurupreet) [reply AdaptiveUiCodelab/gradlew:4]: This is autogenerated and is same for all samples.

File: AdaptiveUiCodelab/README.md (hunk 1 above)
Comment (Gurupreet) [reply AdaptiveUiCodelab/README.md:3]: Sure let me add that and also update the readme headline as we changed the name of codelab

**Pull request merged**